R/designOfExperiments.R

```diff
@@ -128,7 +128,7 @@ designOfExperiments <- function(jaspResults, dataset, options, ...){
                        factors = options[["numberOfFactors"]],
                        runs = runs,
                        resolution = resolution,
-                       # measure = 2^(as.numeric(options[["numberOfFactors"]]) - ),
+                       measure = as.numeric(runs),
```

Comment:
If this column is the same as the `runs` column, it doesn't make much sense to include it. Probably best to remove this column from the table for now. Can you delete this part of the r code, so:
`table$addColumnInfo(name = 'measure', title = gettext("2^(k-p)"), type = 'number')` 
and
`measure = as.numeric(runs),`